fedimint-client/src/module/mod.rs

```diff
@@ -772,6 +781,11 @@ pub trait IClientModule: Debug {
     async fn handle_cli_command(&self, args: &[ffi::OsString])
         -> anyhow::Result<serde_json::Value>;
 
+    async fn handle_rpc(
+        &self,
+        request: serde_json::Value,
+    ) -> BoxStream<'_, anyhow::Result<serde_json::Value>>;
```

Comment:
Why stream? Do we *reaaaly* need support for streaming answers?

Honestly I'm sour on the subscription etc. and here it's not even going through a net layer, so polling (possibly with timeouts for good latency) is practically overhead-free.